tensorflow/lite/micro/micro_interpreter.cc

```diff
@@ -175,6 +206,46 @@ TfLiteStatus MicroInterpreter::PrepareNodeAndRegistrationDataFromFlatbuffer() {
       node->builtin_data = reinterpret_cast<void*>(builtin_data);
       node->custom_initial_data = custom_data;
       node->custom_initial_data_size = custom_data_size;
+
+      // LSTM specific COde
+      size_t alloc_count = subgraph->tensors()->size();
```

Comment:
It might be useful in the design doc to give an overview of what you've found makes the LSTM op different from regular ops, and so what support is required. From the code here it looks like "intermediates" are important, but I'd love to know what you've found about what that means in detail.

Reply:
we try to use similar to TfLite implementation and I also need sometime to understand indepth details